database/schema.sql

```diff
@@ -60,12 +61,21 @@ CREATE TABLE friendMessages (
   timestamp       TIMESTAMP DEFAULT now()
 );
 
+CREATE TABLE notifications (
```

Comment:
So one of the things I had discovered was that the primary keys constraint will not auto increment the id with the 'Not Null' constraint. With the 'Not Null' constraint, when you insert a record/row, it will throw an error unless you provide an insert value for id column as well. 

To get around this for the dailyData table, I had changed the id columns type from 'INT' to 'SERIAL'.